packages/showcases/src/components/common/ExampleQueries.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
```

Comment:
I like the idea of ExampleQueries component.

Could you make the size of query items configurable? I think using smaller examples for e-commerce showcase will be good.

Reference figma node https://www.figma.com/file/Hdr9TuVqvTIS5spOGaayFg/Showcase?node-id=920%3A3648